stone-java-gradle-plugin/src/main/kotlin/com/dropbox/stone/java/StonePlugin.kt

```diff
@@ -52,7 +52,7 @@ public class StonePlugin : Plugin<Project> {
             val mySpecDir: String = specDirPropNameValue ?: "src/${sourceSet.name}/stone"
 
             stoneTask.specDir(mySpecDir)
-            stoneTask.generatorDir("${project.projectDir.absoluteFile}/generator")
+            stoneTask.generatorDir("${project.projectDir.absoluteFile}/generator/java")
```

Comment:
Continues to default to Java.  Put in the json generator in `generator/json` folder.